.stylelintrc

```diff
@@ -0,0 +1,59 @@
+{
+  "rules": {
+    "color-hex-case": "lower",
+    "color-no-invalid-hex": true,
+
+    "function-calc-no-unspaced-operator": true,
+    "function-comma-space-after": "always-single-line",
+    "function-comma-space-before": "never",
+    "function-name-case": "lower",
+    "function-url-quotes": "always",
+    "function-whitespace-after": "always",
+
+    "number-leading-zero": "always",
+    "number-no-trailing-zeros": true,
+    "number-zero-length-no-unit": true,
+
+    "string-no-newline": true,
```

Comment:
Use \\n or \A for a new line not a carriage return
https://github.com/stylelint/stylelint/blob/b1ebde61677799f1a0b3ac33dde75838a31ab67b/src/rules/string-no-newline/README.md